src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2669,10 +2669,16 @@ In addition to the above operations available within ProActive Tasks, users and
 to a specific job via link:/doc/rest/#1218178377[ProActive Signals REST endpoint].
 
 ==== Examples
-Two menus in ProActive *Studio* contain workflow examples and templates, which illustrate the usage of the Signal API, namely:
 
-* The *Signal* menu contains many workflow examples that use all signal operations.
-These workflows use a set of predefined groovy scripts stored in the same bucket. Additionally, we give below an example of a groovy code that uses some Signal API operations.
+===== Standard and simple use with predefined tasks in the Control template menu
+
+  . _Check_For_Signals_: a control pattern that executes a loop until one ore more signals given as input are received.
```

Comment:
one ore more signals given as input => one, or more signals (given as input) is/are received.